### PR TITLE
fix settings and add html extension setting

### PR DESCRIPTION
**Pull request by chuckmah**
## `@ulaval/modul-components`
# PR Checklist

- [x] Provide a small description of the changes introduced by this PR

Changed Settings.json and added html hint plugin to add html formmating options to align with Brio standards

- [x] Include links to issues

https://jira.dti.ulaval.ca/projects/MODUL/issues/MODUL-553

- [ ] Openshift deployment requested

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
+        "**/.vs/": true
+    },
+    "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+    },
+    "html.format.wrapLineLength": 0,
+    "html.format.wrapAttributes": "force-aligned",
+    "typescript.updateImportsOnFileMove.enabled": "prompt"
 }
```
Comment (vidal7) [inline .vscode/settings.json]: stylelint recommande de mettre
"css.validate": false,
    "less.validate": false,
    "scss.validate": false
dans la config pour éviter les erreurs en double
https://github.com/shinnn/vscode-stylelint section optional (but recommended)

File: .vscode/extensions.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
         "eg2.tslint",
         "rbbit.typescript-hero",
```
Comment (vidal7) [inline .vscode/extensions.json]: Tu pourrais retirer typescript-hero étant donné que c'est vs code qui le fait à ta place maintenant avec "editor.codeActionsOnSave": {
        "source.organizeImports": true
    },

File: .vscode/settings.json (hunk 1 above)
Comment (chuckmah) [reply .vscode/settings.json]: done

File: .vscode/extensions.json (hunk 2 above)
Comment (chuckmah) [reply .vscode/extensions.json]: done

**Pull request merged**